src/hotspot/share/oops/cpCache.cpp

```diff
@@ -750,6 +727,8 @@ void ConstantPoolCache::deallocate_contents(ClassLoaderData* data) {
   if (_initial_entries != nullptr) {
```

Comment:
@iklam with moving invokedynamic entries out, do you still need to save initialized entries ?  Does invokehandle need this? (Should have separate RFE if more cleanup is possible)

Reply:
This along with the previous comment about `_invokedynamic_references_map` would probably be better suited for their own RFE. I think the scope of this PR should be limited to the indy structure and its implementation, so any changes related to invokehandle can be traced more easily.